integrationtests/gitcloner/clone_test.go

```diff
@@ -26,6 +26,7 @@ import (
 	cp "github.com/otiai10/copy"
 	"github.com/testcontainers/testcontainers-go"
 	"golang.org/x/crypto/ssh"
+	"golang.org/x/crypto/ssh/knownhosts"
```

Comment:
Good to have this test for the git cloner, however it's unrelated to the change in charts/fleet/templates/deployment_gitjob.yaml